context.md

```diff
@@ -291,14 +330,11 @@ sayHello.call(person, "blue");
 sayHello.call(cat, "peachpuff");
 ```
 
-`apply` works almost exactly like `call`, only you pass in *array* of arguments
-instead of a comma-separated list.
+`apply` works almost exactly like `call`, only you pass in *array* of arguments instead of a comma-separated list.
 
-`apply` is useful when the number of arguments to pass to the function is unknown
-and/or arbitrary.
+`apply` is useful when the number of arguments to pass to the function is unknown and/or arbitrary.
```

Comment:
No idea. Left over from when Adam taught. Pretty much this whole lesson plan is. 😛